ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Raw.scala

```diff
@@ -5,20 +5,62 @@ package com.daml.ledger.participant.state.kvutils
 
 import com.google.protobuf.ByteString
 
+import scala.language.implicitConversions
+
 object Raw {
 
-  sealed trait Bytes {
+  trait Bytes {
     def bytes: ByteString
 
     final def size: Long = bytes.size.toLong
   }
 
+  object Bytes {
+
+    /** This implicit conversion exists to aid in migration.
+      * It will be deprecated and subsequently removed in the future.
+      */
+    implicit def `ByteString to Raw.Bytes`(bytes: ByteString): Bytes = Unknown(bytes)
```

Comment:
Wouldn't we want to use @deprecated annotation for these?

Reply:
Yes, but I was thinking of doing that for 1.10, and removing them in 1.11.

Reply:
Do you think we should do this now?

Reply:
Yes, I'd prefer adding the @deprecated annotations now so that we fix a timeline for actually removing these.

Reply:
I would like to at least finalize the types here. Currently I don't think `Raw.Key` and `Raw.Value` are good enough. I'd like to see (at least) `Raw.LogEntryId`, `Raw.StateKey`, and `Raw.Envelope` instead. There might be more that I'm missing.

Until then, I'd prefer to keep the implicit conversions so that users don't have to migrate twice.

Reply:
Follow-up PR at #8849.